.travis.yml

```diff
@@ -107,7 +107,6 @@ jobs:
       # env: COMPILER=g++-5 SAN_FLAGS="-fsanitize=undefined -fno-sanitize-recover -fno-omit-frame-pointer"
       env:
         - COMPILER="ccache /usr/bin/g++-5"
-        - EXTRA_CXXFLAGS="-D_GLIBCXX_DEBUG"
```

Comment:
Note that `EXTRA_CXXFLAGS` also applied to the unit and regression tests in our Travis config. You are not currently doing so for the buildspec. That might actually be the explanation of the error, given the particular tests that are failing.